bigquery_etl/shredder/amplitude.py

```diff
@@ -85,11 +88,20 @@ def main():
         ),
     )
 
+    conditions = ""
+    if args.submission_date is not None:
+        conditions += f"DATE(submission_timestamp) = DATE '{args.submission_date}'"
+    if args.where is not None:
+        if conditions:
+            conditions += " AND "
+        conditions += args.where
+    where_clause = f" WHERE {conditions}" if conditions else ""
```

Comment:
I'd find this block cleaner if `conditions` were a list that we appended to and then add ANDs at the end:

```suggestion
    where_clause = f" WHERE {' AND '.join(conditions)}" if conditions else ""
```

Reply:
would fix, but this change is being replaced by a derived table.